extensions/airpods-noise-control/src/airpods-menu.ts

```diff
@@ -61,35 +83,55 @@ tell application "System Events"
 					click expandToggle
 					delay 1
 				end if
-				set currentMode to value of checkbox ANCIndex of btMenu as boolean
+				set currentMode to value of checkbox IndexOne of btMenu as boolean
 				if currentMode is true then
-					click checkbox TransparencyIndex of btMenu
-					set setting to "🟢 Transparency"
+					click checkbox IndexTwo of btMenu
+					set output to "🟢 " & OptionTwo
 				else
-					click checkbox ANCIndex of btMenu
-					set setting to "🔵 Noise Cancellation"
+					click checkbox IndexOne of btMenu
+					set output to "🔵 " & OptionOne
 				end if
 			else
 				tell menuBar to click
 				if soundWindowIndex is not equal to -1 then
 					tell menuBar to click
 				end if
-				return "⚠️ Error: AirPods not connected"
+				return "airpods-not-connected"
 			end if
 			tell menuBar to click
 			if soundWindowIndex is not equal to -1 then
 				tell menuBar to click
 			end if
-			return setting
+			return output
 		on error
-			return "⚠️ Error: Control Center not found (check localization)"
+			tell menuBar to click
+			if soundWindowIndex is not equal to -1 then
+				tell menuBar to click
+			end if
+			return "control-center-not-found"
 		end try
 	end tell
 end tell
-`;
+  `;
 
-export function runScript(): Promise<string> {
-  const preferences = getPreferenceValues<Prefs>();
-  const script = getScript(preferences);
-  return runAppleScript(script);
+  const res = await runAppleScript<string>(script);
+  switch (res) {
+    case "sound-not-found": {
+      showFailureToast("", { title: "Sound not found. Check Localization!" });
+      return null;
+    }
+    case "control-center-not-found": {
+      showFailureToast("", {
+        title: "Coltrol Center not found. Check Localization!",
+      });
+      return null;
+    }
+    case "airpods-not-connected": {
+      showFailureToast("", { title: "AirPods not connected!" });
+      return null;
+    }
+    default: {
+      return res;
+    }
+  }
```

Comment:
@pernielsentikaer  There is some issue with `switch` indentation. When running `ray lint --fix` it sets up correct indentation for `switch`. But, when running only `ray lint` it shows the same error as CI.

Reply:
Should be better now